transmogrifier/sources/ead.py

```diff
@@ -138,189 +43,353 @@ def get_optional_fields(self, xml: Tag) -> dict:
+                            timdex.Date(
+                                value=date_value,
+                                kind=date_elem.get("type"),
+                            )
+                        )
+
+                # identifiers
+                for id_value in [
+                    id
+                    for id_elem in collection_description_did.find_all("unitid")
+                    if (id := " ".join(string for string in id_elem.stripped_strings))
+                ]:
+                    fields.setdefault("identifiers", []).append(
+                        timdex.Identifier(
+                            value=id_value,
+                        )
+                    )
+
+                # physical_description
+                if physical_description_value := (
+                    " ".join(
+                        string
+                        for physdesc_elem in [
+                            physdesc_elem
+                            for physdesc_elem in collection_description_did.find_all(
+                                "physdesc", recursive=False
+                            )
+                        ]
+                        for string in physdesc_elem.stripped_strings
+                        if string
+                    )
+                ):
+                    fields["physical_description"] = physical_description_value
+
+                # summary
+                if abstract_value := [
+                    abstract
+                    for abstract_elem in collection_description_did.find_all(
+                        "abstract", recursive=False
+                    )
+                    if (
+                        abstract := " ".join(
+                            string for string in abstract_elem.stripped_strings
+                        )
+                    )
+                ]:
+                    fields["summary"] = abstract_value
 
         return fields
 
+    @classmethod
+    def create_string_from_mixed_value(cls, xml_element: Tag) -> str:
+        """
+        Create a string from an XML element value that contains a mix of strings
+        and XML elements. This method is used for fields where .stripped_strings
+        pulls in unnecessary formatting.
+
+        Args:
+            xml_element: An XML element that may contain a value consisting of
+            strings and XML elements.
+
+        """
+        return " ".join(
+            [
+                string
+                for contents_child in xml_element.contents
+                for string in contents_child.stripped_strings
+                if type(contents_child) == Tag and contents_child.name != "head"
+                if string
+            ]
+        )
+
+    @classmethod
+    def crosswalk_type_value(cls, type_value: str) -> str:
+        """
+        Crosswalk type code to human-readable label.
+
+        Args:
+            type_value: A type value to be crosswalked.
+        """
+        type_crosswalk = {
```

Comment:
Not sure if this rises to the level of `config.py` but  I'm happy to move it there

Reply:
It's definition configuration. I would make it a separate json file, put it in a new config folder, and load it as a new config function.

Reply:
A generic "load JSON file" function or specific function for the type crosswalk?

Reply:
Maybe a generic `load_external_config()` function that can load from any JSON in the config folder, as I expect we'll eventually have language and possibly other config files there as well.